tests/serverpod_test_server/test_integration/column_operations/column_bool_test.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+
+class ColumnBoolEndpoint extends Endpoint {
```

Comment:
Question: This looks wrong, there shouldn't be an endpoint here right? Guessing this is residue from when you migrated the tests?